course_discovery/apps/course_metadata/models.py

```diff
@@ -1275,6 +1276,10 @@ class CourseRun(DraftModelMixin, CachedMixin, TimeStampedModel):
 
     invite_only = models.BooleanField(default=False)
     featured = models.BooleanField(default=False)
+    is_marketing_price = models.BooleanField(default=False, verbose_name=_('Price'), help_text=_(
+        'Indicates whether the course on marketing site is paid'))
```

Comment:
`Indicates whether the course on marketing site is marked paid`

Reply:
Done